### ISSUE TITLE
Packaged Versions (Apps)

### ISSUE THREAD

**Issue by aitorllj93** [prose/prose#813]
Hi, I recently discovered this nice tool. Havent' surfed throught the code yet, but there is something that I miss about it and where I could help you if you want

• Design
  A minimalistic Material Design responsive UI (thinking about something like Jotterpad or Journey for Android)

• Android & ios
  Improved web-app behaviour
  Compiled version if possible (Cordova)

• Desktop
  Chrome App

What do you think about this?

**Comment by erlend-sh** [comment]
:+1: 

Afaik the project is without direct leadership at the moment and is [looking for maintainers](https://github.com/prose/prose/issues/743). In other words, feel free ;)

**Comment by mikemorris** [comment]
/cc @tristen for opinions on "material design" - I'm guessing you're referring to http://www.google.com/design/spec/material-design/introduction.html ? I'm not particularly seeing a significant benefit from revamping the UI at this time.

:+1: on web app improvements (@enochappathurai [cleaned up some stuff on iOS already](https://github.com/prose/prose/issues/803), we just need to deploy it)

:+1: to cordova wrapper as a separate project

:+1: to chrome app, i swear somebody else was working on a firefox app implementation, but I can't find the issue/PR at the moment

**Comment by tristen** [comment]
Yeah, I'm much more interested in talking about specific UI improvements. "Material design" doesn't really mean anything. 

fyi There are much larger [mobile issues](https://github.com/prose/prose/issues/556) to discuss first.
